app/services/db/doc_auth_log/drop_off_rates_helper.rb

```diff
@@ -31,7 +31,6 @@ def select_counts_from_doc_auth_logs
           count(COALESCE(back_image_view_at,mobile_back_image_view_at,capture_mobile_back_image_view_at,present_cac_view_at)) as back_image,
           count(COALESCE(ssn_view_at,enter_info_view_at)) as ssn,
           count(verify_view_at) as verify_info,
-          count(COALESCE(doc_success_view_at,success_view_at)) as doc_success,
```

Comment:
Do we know if any of our reporting consumers rely on this column? Or rely on a specified # of columns?

If so, should we put a 0, -1 or "N/A" there?

Reply:
> Do we know if any of our reporting consumers rely on this column? Or rely on a specified # of columns?
> 
> If so, should we put a 0, -1 or "N/A" there?

I'm not totally familiar with how these are used, so I'd defer to others for specifically how they're used. I wasn't able to find many resources on how these reports are used. Do you know if any exist?

Though, as I look at it again, this may need to stay regardless, because `success_view_at` corresponds to the [CAC flow success step](https://github.com/18F/identity-idp/blob/master/app/services/idv/steps/cac/success_step.rb), which is not intended to be removed with these changes†.

So it might instead need to stay as...

```suggestion
          count(success_view_at) as doc_success,
```

---

† It seems to serve a very similar purpose to the doc auth success step though, so if we're removing one, we should probably remove the other as well. Not something I'd expect to be covered by the current incarnation of the ticket.

Reply:
Restored this for `success_view_at` in 0857bcc, so we're only disregarding the now-removed `doc_success_view_at` column.

Could still do for a second set of eyes (@stevegsa or yourself) to confirm though.

Reply:
Unfortunately, I don't have a good sense of how these are used, Chris and Silke had workflows for these but they've left the project so I don't know who looks at them now 😬

Reply:
The metrics will still work for the other steps after doc_success so it's ok to take it out in both instances.  I don't think there are any queries where we specifically check doc_success other than just reporting it.  After this change we can just remove that step in the funnel.  Then the people that make it to phone screen should still be the same but it would necessarily imply that they succeeded the verify step.

Reply:
Thanks @stevegsa . If I read correctly, it sounds like we'll want to (or are at least covered in reporting) to remove both the `DocSuccessStep` and `Cac::Success` steps. Since this ticket only accounted for the first of these, I'll plan to keep everything as-is for the CAC success step, and open a ticket to explore its removal in the future (UX feedback, etc).

Let me know if I've misunderstood or if there are any concerns to do this separately.